javascript-modules-library/src/components/AbsoluteArea.tsx

```diff
@@ -59,24 +44,13 @@ export function AbsoluteArea({
    * @default false
    */
   readOnly?: boolean | "children";
-  /**
-   * Node type to be used to create the area (if the node does not exist yet)
-   *
-   * @deprecated Use {@link #nodeType} instead
-   * @default jnt:contentList
-   */
-  areaType?: string;
   /**
    * Node type to be used to create the area (if the node does not exist yet)
    *
    * @default jnt:contentList
    */
   nodeType?: string;
-  /**
-   * Map of custom parameters that can be passed to the backend engine for advanced logic.
-   *
-   * @deprecated Not recommended
-   */
+  /** Map of custom parameters that can be passed to the backend engine for advanced logic. */
```

Comment:
So it's no longer not recommended?

Reply:
We have not implemented an alternative and it enables JS/Jsp interop. I can leave it as deprecated but I can't remove it

Reply:
I just recall we wanted to discourage developers to use it. But you're right that without a clear alternative, it's tricky...